corelib/src/odometry/OdometryCuVSLAM.cpp

```diff
@@ -0,0 +1,972 @@
+    camera_rig->num_cameras = cuvslam_camera_objects.size();
+    
+    // Create configuration using Isaac ROS pattern
+    CUVSLAM_Pose cuvslam_imu_pose;
+    // if (!data.imu().empty()) {
+    //     // Get IMU pose relative to camera (base frame)
+    //     // data.imu().localTransform() gives us camera_pose_imu (camera in IMU frame)
+    //     // We need imu_pose_camera (IMU in camera frame) for cuVSLAM
+    //     Transform imu_pose_camera = data.imu().localTransform().inverse();
+    //     cuvslam_imu_pose = convertCameraPoseToCuVSLAM(imu_pose_camera);
+    // } else {
+    //     // Set identity pose if no IMU
+    //     cuvslam_imu_pose = convertCameraPoseToCuVSLAM(Transform::getIdentity());
+    // }
+    
+    // No IMU for now, use identity pose
+    cuvslam_imu_pose = convertImuPoseToCuVSLAM(Transform::getIdentity());
+
+    
+    CUVSLAM_Configuration * configuration = CreateConfiguration(&cuvslam_imu_pose);
+    
+    // Enable IMU fusion if we have IMU data
+    if (!data.imu().empty()) {
+        configuration->enable_imu_fusion = 1;
+    }
+    
+    // Create tracker
+    CUVSLAM_TrackerHandle tracker_handle;
+    CUVSLAM_Status status = CUVSLAM_CreateTracker(&tracker_handle, camera_rig, configuration);
+    cuvslam_handle = tracker_handle;
+
+    delete configuration;
+    delete camera_rig;
+
+    if(status != CUVSLAM_SUCCESS)
+    {
+        UERROR("Failed to create cuVSLAM tracker: %d", status);
+        return false;
+    }
+    
+    return true;
+}
+
+/*
+Implementation based on Isaac ROS VisualSlamNode::VisualSlamImpl::CreateConfiguration()
+Source: isaac_ros_visual_slam/isaac_ros_visual_slam/src/impl/visual_slam_impl.cpp:379-422
+https://github.com/NVIDIA-ISAAC-ROS/isaac_ros_visual_slam/blob/19be8c781a55dee9cfbe9f097adca3986638feb1/isaac_ros_visual_slam/src/impl/visual_slam_impl.cpp#L379-L422    
+*/
+CUVSLAM_Configuration * CreateConfiguration(const CUVSLAM_Pose * cv_base_link_pose_cv_imu)
+{
+    // Note: cv_base_link_pose_cv_imu parameter is not used in current implementation
+    // but kept for future IMU integration
+    (void)cv_base_link_pose_cv_imu; // Suppress unused parameter warning
+    CUVSLAM_Configuration * configuration = new CUVSLAM_Configuration();
+    CUVSLAM_InitDefaultConfiguration(configuration);
+    
+    // Core Visual Odometry Settings 
+    configuration->multicam_mode = 0;                    // Single camera setup
+    configuration->use_motion_model = 1;                 // Enable motion model for better tracking
+    configuration->use_denoising = 0;                    // Disable denoising by default
+    configuration->use_gpu = 1;                          // Use GPU acceleration
+    configuration->horizontal_stereo_camera = 1;         // Stereo camera configuration
+    
+    // SLAM Disabled (VO-only mode) 
+    configuration->enable_localization_n_mapping = 0;    // Disable SLAM - VO only
+    configuration->enable_observations_export = 0;       // Not needed for VO
```

Comment:
Could be interesting to enable this to see features that are used in the odometry view. You would then need to fill the `info->newCorners` member in the `OdometryInfo` object in `computeTransform()`.